tcms/templates/include/tp_toolbar.html

```diff
@@ -0,0 +1,135 @@
+            </button>
+        </div>
+    </div>
+    <div class="toolbar-separator">
+        <div class="dropdown btn-group">
+            <button type="button" class="btn btn-default dropdown-toggle" id="filter-button" data-toggle="dropdown"
+                    aria-haspopup="true" aria-expanded="false">Priority<span class="caret"></span></button>
+            <ul class="dropdown-menu">
+                <li class="selected"><a href="#">P1</a></li>
+                <li><a href="#">P2</a></li>
+                <li><a href="#">P3</a></li>
+                <li><a href="#">P4</a></li>
+                <li><a href="#">P5</a></li>
+            </ul>
+        </div>
+        <div class="dropdown btn-group">
+            <button type="button" class="btn btn-default dropdown-toggle" id="filter-button" data-toggle="dropdown"
+                    aria-haspopup="true" aria-expanded="false">Status<span class="caret"></span></button>
+            <ul class="dropdown-menu">
+                <li class="selected"><a href="#">PROPOSED</a></li>
+                <li><a href="#">CONFIRMED</a></li>
+                <li><a href="#">DISABLED</a></li>
+                <li><a href="#">NEED UPDATE</a></li>
+            </ul>
+        </div>
+        <button class="btn btn-default" type="button">Default tester</button>
+        <button class="btn btn-default" type="button">Delete</button>
+    </div>
+    <div class="toolbar-pf-find toolbar-search">
+        <button class="btn btn-link btn-find" type="button">
+            <span class="fa fa-search"></span>
+        </button>
+        <div class="find-pf-dropdown-container">
+            <input type="text" class="form-control" id="find" placeholder="Find By Keyword...">
+            <div class="find-pf-buttons">
+                {#                todo should be populated by the js#}
+                <span class="find-pf-nums">1 of 1</span>
+                <button class="btn btn-link" type="button">
+                    <span class="fa fa-angle-up"></span>
+                </button>
+                <button class="btn btn-link" type="button">
+                    <span class="fa fa-angle-down"></span>
+                </button>
+                <button class="btn btn-link btn-find-close" type="button">
+                    <span class="pficon pficon-close"></span>
+                </button>
+            </div>
+        </div>
+    </div>
+</div>
+
+<script>
+    {#    todo move to js file #}
+    (function($) {
+        $(document).ready(function() {
+            // Upon clicking the find button, show the find dropdown content
+            $(".btn-find").click(function() {
+                $(this).parent().find(".find-pf-dropdown-container").toggle();
+            });
+            // Upon clicking the find close button, hide the find dropdown content
+            $(".btn-find-close").click(function() {
+                $(".find-pf-dropdown-container").hide();
+            });
+        });
+    })(jQuery);
```

Comment:
I don't think we need this find button so let's remove it and focus on the things we know that we need ATM.